diff -ruN cppqed-blitz/blitz/generate/genstencils.py cppqed-blitz.new/blitz/generate/genstencils.py
--- cppqed-blitz/blitz/generate/genstencils.py	2014-04-04 03:46:00.455290000 +0200
+++ cppqed-blitz.new/blitz/generate/genstencils.py	2014-04-04 03:55:00.974195000 +0200
@@ -6,7 +6,7 @@
 import sys
 
 os=open(sys.argv[1],'w')
-print("Generating file {}".format(sys.argv[1]))
+print("Generating file {0}".format(sys.argv[1]))
 
 def BZ_ET_STENCIL_REDIRECT(name):
     stub="""
