diff -r de47c97201fb lib/Makefile.am
--- a/lib/Makefile.am	Fri Feb 14 19:14:01 2014 +0100
+++ b/lib/Makefile.am	Sun Mar 02 14:22:24 2014 +0100
@@ -6,9 +6,9 @@
 
 AM_CPPFLAGS = -I$(top_srcdir) -I$(top_builddir) $(BOOST_CPPFLAGS)
 
-lib_LTLIBRARIES = libblitz.la
-libblitz_la_SOURCES = $(top_srcdir)/src/globals.cpp
-libblitz_la_LIBADD = $(LIBS) $(CXX_LIBS)
+lib_LTLIBRARIES = libblitzcppqed.la
+libblitzcppqed_la_SOURCES = $(top_srcdir)/src/globals.cpp
+libblitzcppqed_la_LIBADD = $(LIBS) $(CXX_LIBS)
 
 
 clean-local:
